runtime/lua/vim/keymap.lua

```diff
@@ -89,7 +94,7 @@ end
 --- ```
 ---
 ---@param opts table|nil A table of optional arguments:
----                      - "buffer": (number|boolean) Remove a mapping from the given buffer.
+---                      - "buffer": (integer|boolean) Remove a mapping from the given buffer.
```

Comment:
Why is this only changed for `vim.keymap.del()` but not `vim.keymap.set()`?

Reply:
No reason in particular. I was just fixing warnings and there were none for this method, but I'll make the change as well.